tests/payloads/github.headers.txt

```diff
@@ -0,0 +1,7 @@
+Request URL: https://openedx-webhooks.herokuapp.com/github/pr
```

Comment:
These are place holders for now. It was needed for me to reference as I unpack the file. We may use these as fixtures later. Or we may not, I'm not sure yet. But this is the raw payload that GitHub sends us.